### PR TITLE
Mar 9 crud annonces de repas categories

### PR DESCRIPTION
**Pull request by AbderrahimAgra**

### CODE REVIEW

File: backend/controllers/userController/managerController.js [hunk 1]
```diff
@@ -18,6 +19,74 @@ const managerUser = async (req, res) => {
...
+    const data =  await User.findOneAndUpdate({_id:id},{isBanned: true})
+    if(!data) res.send('not')
+    res.send('updated')
+
+}
+
+
+
 module.exports = {
-  managerUser
```
Comment (elmehdinainia) [inline backend/controllers/userController/managerController.js:22]: delet this line

File: backend/models/category.js [hunk 2]
```diff
@@ -2,13 +2,13 @@ const mongoose = require('mongoose')
 
 const categorySchema= mongoose.Schema({
 
-    name: [
```
Comment (elmehdinainia) [inline backend/models/category.js:5]: delet this line

File: backend/models/category.js [hunk 3]
```diff
@@ -2,13 +2,13 @@ const mongoose = require('mongoose')
...
+    name: 
         {
             type: String,
             required: true,
             trim: true
         }
-      ]
```
Comment (elmehdinainia) [inline backend/models/category.js:11]: add this line

File: backend/routes/UserRouter/managerRouter.js [hunk 4]
```diff
@@ -1,13 +1,24 @@
 const router = require('express').Router()
 const {
-  managerUser
```
Comment (elmehdinainia) [inline backend/routes/UserRouter/managerRouter.js:3]: fix this

**Review by elmehdinainia — CHANGES_REQUESTED** [review]

**Review by elmehdinainia — APPROVED** [review]

**Comment by AbderrahimAgra** [comment]
bug are fixed

**Pull request merged**